libnavui-app/src/main/java/com/mapbox/navigation/ui/app/internal/routefetch/RoutesState.kt

```diff
@@ -18,20 +18,20 @@ sealed class RoutesState {
      * Represents the state when the route is being fetched.
      * @param requestId of the route requested
      */
-    data class Fetching internal constructor(val requestId: Long) : RoutesState()
+    data class Fetching constructor(val requestId: Long) : RoutesState()
```

Comment:
Do you only need to remove the internal modifier for tests? You can mock the objects instead. Just wondering why you chose this approach.

Reply:
This is internal package, so it doesn't really matter.